pkg/customresourcestate/registry_factory.go

```diff
@@ -635,6 +635,12 @@ func toFloat64(value interface{}, nilIsZero bool) (float64, error) {
 		}
 		return 0, nil
 	case string:
+		if value.(string) == "True" || value.(string) == "true" || value.(string) == "Yes" || value.(string) == "yes" {
+			return 1, nil
+		}
+		if value.(string) == "False" || value.(string) == "false" || value.(string) == "No" || value.(string) == "no" {
+			return 0, nil
+		}
```

Comment:
Let's normalize the string first so we catch also different capitalizations.

```suggestion
                normalized := strings.ToLower(value.(string))
		if normalized == "true" || normalized == "yes" {
			return 1, nil
		}
		if normalized == "false" || normalized == "no" {
			return 0, nil
		}
```

Reply:
Good one. Thanks!